src/components/software-registry/cards/mainCard.tsx

```diff
@@ -217,7 +217,7 @@ const MainCard: React.FC<CardProps> = ({
    */
   const renderTags = () => (
     <Box>
-      {tags.map((tag) => (
+      {tags.slice(0, 3).map((tag) => (
```

Comment:
So we only show the first 3 tags here, but the rest are shown on the details view?
If so, then perhaps we should render some kind of indicator that there are more tags? (+)